src/Resources/contao/library/Contao/User.php

```diff
@@ -357,17 +357,14 @@ public function authenticate()
 	 */
 	public function login()
 	{
+		dump(\Input::post('username'));
```

Comment:
@leofeyer ?

Reply:
It's not completed yet 😀

Reply:
Ah fuck, sorry. I thought this was a 4.4.5 commit, sorry about that!